compose.yml

```diff
@@ -29,13 +25,6 @@ services:
     ports:
       - "8005:80"
   nginx4:
-    image: nginx:latest
-    ports:
-      - "8004:80"
-
-  nginx5:
-    image: nginx:latest
```

Comment:
@s7solomon  i see you made a very good Dockerfile

Reply:
but need a little modification